pkg/subctl/cmd/gather/connectivity.go

```diff
@@ -24,7 +24,7 @@ import (
 
 const (
 	gatewayPodLabel             = "app=submariner-gateway"
-	routeagentPodLabel          = "app=submariner-routeagent"
+	RouteagentPodLabel          = "app=submariner-routeagent"
```

Comment:
Why was this exported? I don't see it used outside the package.

Reply:
Missed this cleanup after some refactoring.